text/0000-named-tracers.md

```diff
@@ -0,0 +1,91 @@
+# Named Tracers
```

Comment:
I think this logic applies to Meter as well. Consider to make this change to apply to all the "signal producers" factories: metrics, traces, in the future logs.

Reply:
I would very much prefer having dedicated RFCs for metrics and logs. As we have seen it's easy enough to get hung up on details and move forward painfully slow even with the narrow focus we have here. Widening this focus and trying to covers metrics as well with this RFC (or even logs) will not add any benefit. This RFC targets a specific part of the specification, namely tracers. Other specification topics deserve dedicated and thoroughly discussed RFCs as well.